src/gui/SetupDialog.cpp

```diff
@@ -145,7 +145,7 @@ SetupDialog::SetupDialog(ConfigTabs tab_to_open) :
 {
 	setWindowIcon(embed::getIconPixmap("setup_general"));
 	setWindowTitle(tr("Settings"));
-	setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+	setWindowFlag(Qt::WindowContextHelpButtonHint, false);
```

Comment:
From https://doc.qt.io/qt-5/qwidget.html#setWindowFlag:
> This function was introduced in Qt 5.9.

I think it's better to revert this for compatibility and add a TODO comment instead.

Reply:
I didn't notice that Travis CI failed. Good catch! Fixed in 8c5a88c.